HyperPlay-Gaming/hyperplay-desktop-client
# feat(quest): add support for sending completed claim to API

## Summary
After the claim transaction finishes, we need to send the transaction to the API to notify the reward was claimed and the signature was used.

Right now, as long as there is a valid signature, we show the playstreak reward as still not claimed.

## How to test
1. Claim an on-chain reward
2. Double check that the playstreak is reset in the UI after the balance reloads


closes https://github.com/HyperPlay-Gaming/product-management/issues/528

---

Use the following Checklist if you have changed something on the Backend or Frontend:
